### PR TITLE
Add docs for Canny (Actions) Destination

### PR DESCRIPTION
**Pull request by BR4VE**
### Proposed changes

- Added documentation for Canny (Actions) Destination as it is required to get the destination in public beta.

### Merge timing
- ASAP once approved, we'd like to make our destination public.

### CODE REVIEW

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
 ---
-title: 'Canny-Functions Destination'
-hidden: true
-beta: true
+title: Canny (Actions) Destination
+id: <integration_id>
```
Comment (BR4VE) [inline src/connections/destinations/catalog/canny-functions/index.md:3]: I'm not sure what to write for id. I was not able to find the id of our destination in private beta.

**Comment by BR4VE** [comment]
@stayseesong Any updates on this?

File: src/connections/destinations/catalog/canny-functions/index.md (hunk 1 above)
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:3]:
```suggestion
id: 6489bbade6fe3eb57c13bd6a
beta: true
hidden: true
private: true
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Canny](https://canny.io?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a single place for all customer feedback. It saves you time managing all the feedback while keeping your customers in the loop. Let your customers post and vote on feedback from within your website or mobile app. You'll get an organized list of feedback that you can use to inform your roadmap.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:8]:
```suggestion
[Canny](https://canny.io?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a single place for all customer feedback. It saves you time managing all the feedback while keeping your customers in the loop. Let your customers post and vote on feedback from within your website or mobile app. You'll get an organized list of feedback that you can use to inform your roadmap.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+
+This destination is maintained by Canny. For any issues with the destination, [contact the Canny Support team](mailto:segment-help@canny.io).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:10]:
```suggestion
Canny maintains this destination. For any issues with the destination, [contact the Canny Support team](mailto:segment-help@canny.io).
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+
+{% include content/ajs-upgrade.md %}
+
+## Getting started
+
+1. Go to your [Canny Admin Segment Settings](https://canny.io/redirect?to=%2Fadmin%2Fsettings%2Fsegment).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:16]:
```suggestion
1. Go to your [Canny Admin Segment Settings](https://canny.io/redirect?to=%2Fadmin%2Fsettings%2Fsegment){:target="_blank"}.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 5]
```diff
@@ -1,5 +1,55 @@
...
+2. Obtain your API key by installing the integration.
+3. From the Segment web app, click **Catalog**, then click **Destinations**.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:18]:
```suggestion
3. From the Segment web app, navigate to **Connections > Catalog**, then select the **Destinations** tab in the catalog.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 6]
```diff
@@ -1,5 +1,55 @@
...
+4. Find the Destinations Actions item in the left navigation, and click it.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:19]:
```suggestion
4. Search for *Canny (Actions)* and select it.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 7]
```diff
@@ -1,5 +1,55 @@
...
+5. Click **Configure Canny (Actions)**.
+6. Select an existing Source to connect to Canny (Actions).
+7. After you finished configuring the destination, update the API key in the Canny (Actions) destination settings, enable destination and save changes.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:22]:
```suggestion
7. Enter your configuration settings. 
8. Update the API key in the Canny (Actions) destination settings.
9. Enable the destination and save changes.
```

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 8]
```diff
@@ -1,5 +1,55 @@
...
+analytics.group("0e8c78ea9d97a7b8185e8632", {
+  name: "X Corp",
+});
+```
+
+Group calls will be sent to Canny as an group event. Once sent, groups will appear in Canny as companies attached to users. The `name` trait is a **required** field.
+
+### Custom Fields
+
+Custom Fields is a list of [traits](/docs/connections/spec/identify/#traits) to be imported as custom fields for Identify events.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:53]: @BR4VE are you sure you want to include these sections? These sections aren't needed for actions destinations and are usually only needed for the classic docs.

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
Also, not sure if you wanted to add this, but normally our Actions docs has a benefits section. Here's the template: 

## Benefits of <destination_name> (Actions) vs <destination_name> Classic

<destination_name> (Actions) provides the following benefits over the classic <destination_name> destination:

- **Main point 1**. One or two sentences that back up the main point.
- **Main point 2**. One or two sentences that back up the main point.

File: src/connections/destinations/catalog/canny-functions/index.md [hunk 9]
```diff
@@ -1,5 +1,55 @@
...
+title: Canny (Actions) Destination
+id: <integration_id>
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Canny](https://canny.io?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a single place for all customer feedback. It saves you time managing all the feedback while keeping your customers in the loop. Let your customers post and vote on feedback from within your website or mobile app. You'll get an organized list of feedback that you can use to inform your roadmap.
+
+This destination is maintained by Canny. For any issues with the destination, [contact the Canny Support team](mailto:segment-help@canny.io).
+
```
Comment (stayseesong) [inline src/connections/destinations/catalog/canny-functions/index.md:11]:
```suggestion

> success ""
> **Good to know**: This page is about the [Actions-framework](/docs/connections/destinations/actions/) Canny Segment destination. There's also a page about the [non-Actions Canny destination](/docs/connections/destinations/catalog/canny/). Both of these destinations receive data from Segment.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/canny-functions/index.md (hunk 8 above)
Comment (BR4VE) [reply src/connections/destinations/catalog/canny-functions/index.md]: @stayseesong Thanks for mentioning this. I was not sure it it's required or not. Now I removed it 👍

**Comment by BR4VE** [comment]
@stayseesong Updated the PR based on your suggestions. Waiting for your review 👍

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@joe-ayoub-segment Is this one going straight to public beta or will it be in private beta first?

**Comment by joe-ayoub-segment** [comment]
Hi @BR4VE , @stayseesong , 

The docs just go live once they've been approved - @tcgilbert can you confirm please?

**Comment by BR4VE** [comment]
@stayseesong Updated 👍. Waiting for you to review.

**Comment by stayseesong** [comment]
Looks good @BR4VE, I'm going to close this PR out as I created an editable version for the docs team to work on here #4965

**Pull request closed without merging**